client/components/push-notification/prompt.jsx

```diff
@@ -79,13 +80,25 @@ const PushNotificationPrompt = React.createClass( {
 		return <Notice className="push-notification__notice" text={ noticeText } icon="bell" onDismissClick={ this.props.dismissNotice } />;
 	},
 
+	isUnsupportedChromeVersion: function() {
+		if ( global.window && global.window.chrome && global.window.navigator.appVersion ) {
+			const chromeVersion = global.window.navigator.appVersion.match( /Chrome\/(\d+)/ )[ 1 ];
+			if ( chromeVersion < 50 ) {
+				analytics.mc.bumpStat( 'calypso_push_notification_chrome', chromeVersion );
```

Comment:
32-character limit applies, this label is exactly 32 characters. `calypso_` comes from the naming convention. We can try something like `calypso_push_notif_unsup_chrome' (that's 31 characters).